flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCUpsertOutputFormat.java

```diff
@@ -184,15 +186,6 @@ public synchronized void close() throws IOException {
 		if (this.scheduledFuture != null) {
 			scheduledFuture.cancel(false);
 			this.scheduler.shutdown();
```

Comment:
Should we change this to `shutdownNow()`? I think the async flusher can quit now.

Reply:
In this case, I think we should not interrupt thread, we have good way to deal with it now, interrupt may make things complicate.